pkg/p2p/libp2p/libp2p.go

```diff
@@ -35,7 +35,12 @@ import (
 	"github.com/multiformats/go-multistream"
 )
 
-var _ p2p.Service = (*Service)(nil)
+var (
+	_ p2p.Service = (*Service)(nil)
+
+	// ErrBadNetwork indicates that it is suspected that network is currently in bad condition
+	ErrBadNetwork = errors.New("bad network")
```

Comment:
is this used anywhere?

Reply:
Huh I think not. it was there before so I forgot to erase it. Good catch!